profiles/thalesgomest.json

```diff
@@ -0,0 +1,7 @@
+{
+  "image": "https://user-images.githubusercontent.com/97575616/207838744-93e185a3-7959-46d8-82db-f7d69e843ee8.png",
```

Comment:
I recommend more of an image like this. What do you think? 🙂

![image](https://user-images.githubusercontent.com/51391473/207966988-9a959e22-a0cc-4393-a54d-04eda0e241fb.png)

```suggestion
  "image": "https://user-images.githubusercontent.com/51391473/207966988-9a959e22-a0cc-4393-a54d-04eda0e241fb.png",
```